examples/platforms/cc2538/CMakeLists.txt

```diff
@@ -28,13 +28,19 @@
 
 set(OT_PLATFORM_LIB "openthread-cc2538" PARENT_SCOPE)
 
+if(OT_CONFIG MATCHES "none")
+    set(OT_CONFIG "openthread-core-cc2538-config.h")
+    set(OT_CONFIG ${OT_CONFIG} PARENT_SCOPE)
```

Comment:
Note (not request for change): This doesn't change the CACHE variable, so that will still appear as 'none'.

Reply:
I am still learning the cmake, but I think/guess that's the behavior we want (we just want to let platform to override this value in parent scope variable if not given by user through the cache variable).

Reply:
I agree, I think this produces more sensible behaviour than changing the cache directly.